src/main/scala/is/hail/expr/ir/MatrixIR.scala

```diff
@@ -1132,19 +1132,13 @@ case class MatrixMapRows(child: MatrixIR, newRow: IR, newKey: Option[(IndexedSeq
     MatrixMapRows(newChildren(0).asInstanceOf[MatrixIR], newChildren(1).asInstanceOf[IR], newKey)
   }
 
-  val newRVRow = newRow.typ.asInstanceOf[TStruct].fieldOption(MatrixType.entriesIdentifier) match {
-    case Some(f) =>
-      assert(f.typ == child.typ.entryArrayType)
-      newRow
-    case None =>
-      InsertFields(newRow, Seq(
-        MatrixType.entriesIdentifier -> GetField(Ref("va", child.typ.rvRowType), MatrixType.entriesIdentifier)))
-  }
+  val newRVRow = InsertFields(newRow, Seq(
```

Comment:
rebase issue? This was a good change

Reply:
Ugh, yes, rebase issue. Thanks for catching that.